tests/end_to_end/netsh_test.cpp

```diff
@@ -408,12 +410,19 @@ TEST_CASE("show sections cgroup_sock_addr.sys", "[netsh][sections]")
                                         "              hash    56      8     1000  socket_cookie_map\n";
 
     bool output_matches =
-        (output ==
-             std::vformat(
-                 expected_output, std::make_format_args(code_size[0], code_size[1], code_size[2], code_size[3])) ||
+        (output == std::vformat(
+                       expected_output,
+                       std::make_format_args(
+                           code_size[0], code_size[1], code_size[2], code_size[3], code_size[4], code_size[5])) ||
          output == std::vformat(
                        expected_output,
-                       std::make_format_args(old_code_size[0], old_code_size[1], old_code_size[2], old_code_size[3])));
+                       std::make_format_args(
+                           old_code_size[0],
+                           old_code_size[1],
+                           old_code_size[2],
+                           old_code_size[3],
+                           code_size[4],
```

Comment:
not completely fluent in what this is doing - but this is somewhat conspicuous that fiends 4 and 5 are from code_size, not from old_code_size. this is intended?